aiida/orm/importexport.py

```diff
@@ -2254,15 +2263,15 @@ def export_tree(what, folder, allowed_licenses=None, forbidden_licenses=None,
 
     ## Universal "entities" attributed to all types of nodes
     # Logs
-    if include_logs:
+    if include_logs and len(to_be_exported) is not 0:
```

Comment:
Don't check for empty list by comparing length to zero, if it is an empty list, you can just do `if include_logs and to_be_exported:`

Reply:
Done - also removed this part completely from the other PR, when I realized it was duplicated.

Reply:
@sphuber @CasperWA 
This is true for sequences and it the advised way to check if they are empty
https://www.python.org/dev/peps/pep-0008/

`to_be_exported`  is a set, and a set is not a sequence (at least conceptually).
From a fast lookup, I see that the above check does by accident the correct thing
https://stackoverflow.com/questions/44813375/truth-value-of-empty-set
(from the discussion, I understand that it is not documented but since it is based on the implementation of the sequence, it has the 'behaviour' of a sequence - this is what I understand for the second comment).

If it is the case, I would do a comparison based on the size of the set.